daemon/daemon.go

```diff
@@ -405,19 +405,19 @@ func (d *Daemon) init() error {
 	fw.WriteString(common.FmtDefineArray("ROUTER_IP", d.conf.NodeAddress.IPv6Address))
 
 	ipv4GW := d.conf.NodeAddress.IPv4Address
-	fmt.Fprintf(fw, "#define IPV4_GATEWAY %#x\n", binary.LittleEndian.Uint32(ipv4GW))
+	fmt.Fprintf(fw, "#define IPV4_GATEWAY %#x\n", byteorder.NetworkToHostU32(ipv4GW))
```

Comment:
I think above conversion is not correct. BPF code treats this as network byte order, so this would break things.

Take `fmt.Fprintf(fw, "#define LXC_IPV4 %#x\n", binary.BigEndian.Uint32(e.IPv4))`, for example. That is stored in host order, hence `bpf_htonl()` is needed in BPF code.

I think this `binary.BigEndian` interface is highly confusing.